<launch>

    <node name="controller_node" pkg="pid_pkg" type="pid_node" output="screen">
        <rosparam command="load" file="$(find pid_pkg)/config/default.yaml" />
    </node>

</launch>